docs/components/modeler/web-modeler/process-application-versioning.md

```diff
@@ -1,45 +1,41 @@
 
-- When you create a version in any process application file, a version is also created with the same name (version tag) for all files currently available in the process application.
+- When you create a process application version, the input version tag is applied to the `versionTag` field in the main process XML.
+- This helps you track a process application throughout its development lifecycle and ensures that the correct version is referenced.
 
-- This allows you to track a process application through the development lifecycle and ensure the correct version is called.
+## Version creation
 
-## A worked example
+To create a process application version:
 
-In this example, you have a process application that contains multiple files as well as the main process.
+1. Open any file in a process application, click on the **Create version** button in the **Versions** dropdown menu.
+2. Enter a **Version tag** in the version creation modal.
+3. Finish by clicking on the **Create** button.
 
-- You create a new version for the main process and enter "1.2" as the version tag.
-- When you create this new version, a version is also created for every other file and named with the same "1.2" version tag.
-- All files in the process application now have a "1.2" version.
+![Create a version menu](img/versions/web-modeler-version-process-application-versions-menu.png)
 
-![Diagram showing an example process application bulk version creation](img/process-applications/process-application-version-diagram.png)
+![Create a version modal](img/versions/web-modeler-version-create-process-application-version.png)
 
-## Create a bulk version
+Now when you open the versions page (use the **Show versions** button in the **Versions** dropdown menu), you can use the file navigation header buttons to switch between files and view their content. See also [compare versions](/components/modeler/web-modeler/versions.md#compare-versions).
 
-To create a bulk version for a process application:
+![Versions list with file navigation header buttons highlighted](img/versions/web-modeler-version-view-process-application-version.png)
 
-1. Open any file in a process application and [create a version](versions.md#create-a-version).
-1. Enter a **Version tag**.
-1. Select **Create** to create the version for every file in the process application.
+## Version actions
 
-![Create a version screen](img/versions/web-modeler-version-create-versioned-milestone-highlight.png)
+You can perform the following actions on a process application version:
 
-Now when you open the versions list for any file in the process application, you can use the file navigation header buttons to switch between files, and view their individual versions list. See [compare versions](/components/modeler/web-modeler/versions.md#compare-versions).
+1. Restore as latest (restores a version as the current state of the process application).
```

Comment:
```suggestion
1. Restore as latest, useful for reverting changes, making further edits, git syncing, downloading, or using Play.
```
(S) Git sync can be linked to https://docs.camunda.io/docs/next/components/modeler/web-modeler/git-sync/